get_goes_x.py

```diff
@@ -5,18 +5,20 @@
 import argparse
 import tables
 import time
+import json
 
 import numpy as np
 from astropy.io import ascii
+from astropy.table import Table, join
 from Chandra.Time import DateTime
 
 parser = argparse.ArgumentParser(description='Get GOES_X data')
 parser.add_argument('--h5',
-                    default='GOES_X.h5',
+                    default='GOES_X_16.h5',
```

Comment:
Why this change to 16?  What happens when they go to 17?  Is the new file format incompatible with the existing GOES_X.h5?

Reply:
I figured we were better off with a new file thanks to a totally different data source and different columns.

Reply:
Do we have an expectation to use the GOES data saved in the replan central project for other analysis?  I couldn't figure out why it went back to like 2013 just to make the replan central plots, but if the point is that it should be a resource, that's a requirement I had not considered.

Reply:
Yes it is a resource because AFAIK it is exceedingly difficult to get full-resolution archival GOES data.  So I decided to start collecting it locally.